app/src/main/java/ie/macinnes/tvheadend/player/ExoPlayerUtils.java

```diff
@@ -91,30 +94,30 @@ public static TvTrackInfo buildTvTrackInfo(Format format) {
     }
 
     // Track name construction.
-    private static String buildTrackName(Format format) {
+    private static String buildTrackName(Format format, Context context) {
         String trackName;
         if (MimeTypes.isVideo(format.sampleMimeType)) {
-            trackName = joinWithSeparator(joinWithSeparator(buildResolutionString(format),
-                    buildBitrateString(format)), buildTrackIdString(format));
+            trackName = joinWithSeparator(joinWithSeparator(buildResolutionString(format, context),
+                    buildBitrateString(format)), buildTrackIdString(format, context));
         } else if (MimeTypes.isAudio(format.sampleMimeType)) {
             trackName = joinWithSeparator(joinWithSeparator(joinWithSeparator(buildLanguageString(format),
-                    buildAudioPropertyString(format)), buildBitrateString(format)),
-                    buildTrackIdString(format));
+                    buildAudioPropertyString(format, context)), buildBitrateString(format)),
+                    buildTrackIdString(format, context));
         } else {
             trackName = joinWithSeparator(joinWithSeparator(buildLanguageString(format),
-                    buildBitrateString(format)), buildTrackIdString(format));
+                    buildBitrateString(format)), buildTrackIdString(format, context));
         }
-        return trackName.length() == 0 ? "unknown" : trackName;
+        return trackName.length() == 0 ? context.getString(R.string.track_name_unknown) : trackName;
     }
 
-    private static String buildResolutionString(Format format) {
+    private static String buildResolutionString(Format format, Context context) {
         return format.width == Format.NO_VALUE || format.height == Format.NO_VALUE
-                ? "" : format.width + "x" + format.height;
+                ? "" : format.width + context.getString(R.string.resolution_separator) + format.height;
```

Comment:
Possibly, though I'm not entirely sure. I felt it was best to add everything, and at worst they can be set as untranslatable in the strings.xml file